### ISSUE TITLE
No Charts available from Cluster Tools option when mcm is disabled

### ISSUE THREAD

**Issue by nwmac** [rancher/rancher#33688]
Deploy Rancher with the multi-cluster-manager flag disable. In the UI, go to Charts - there are no charts available.

Go to Chart Repositories and all 3 are stuck 'In Progress'.

Looking at the log (running in Docker), I see:

```
E0720 08:31:20.616613      32 reflector.go:139] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: Failed to watch *v3.Project: failed to list *v3.Project: the server could not find the requested resource (get projects.meta.k8s.io)
E0720 08:32:13.633740      32 reflector.go:139] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: Failed to watch *v3.Project: failed to list *v3.Project: the server could not find the requested resource (get projects.meta.k8s.io)
E0720 08:33:00.476250      32 reflector.go:139] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: Failed to watch *v3.Project: failed to list *v3.Project: the server could not find the requested resource (get projects.meta.k8s.io)
E0720 08:33:35.738712      32 reflector.go:139] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: Failed to watch *v3.Project: failed to list *v3.Project: the server could not find the requested resource (get projects.meta.k8s.io)
E0720 08:34:17.223526      32 reflector.go:139] pkg/mod/github.com/rancher/client-go@v0.21.0-rancher.1/tools/cache/reflector.go:168: Failed to watch *v3.Project: failed to list *v3.Project: the server could not find the requested resource (get projects.meta.k8s.io)
```

Looks like its trying to watch project resources, which don't exist with MCM disabled - I don't know if this is blocking the chart repository sync.

**Note:**
Happens on a HA on the downstream clusters, when MCM is enabled also.

**Comment by kravciak** [comment]
I have all repos stack `In Progress` on master-head (b6bcfad) even with multi-cluster-management active (and no such error in docker log).

**Comment by StrongMonkey** [comment]
This is duplicate issue of https://github.com/rancher/rancher/issues/33711. Can be retested or closed.

**Comment by catastrophe-brandon** [comment]
When installed via Docker (commit [d8824a1](https://github.com/rancher/rancher/commit/d8824a19240d4acb30207bfbd1d293189add0de4)), I was able to see the Charts and the Chart Repositories in the UI. All the repositories were `Active`. However, I noticed a number of 404s surfacing in the dev console.

[riker.xyz-charts.log](https://zube.io/files/rancher/c9fa3bc4b067056a7bc710d33b8e8d14-riker-xyz-charts.log)
